string ADD="add"
string SUBTRACT="subtract"

string operation
string umrf_json
